shared/domains/chelonia/SPMessage.js

```diff
@@ -290,7 +290,7 @@ export class SPMessage {
     if (!state?._vm?.authorizedKeys && head.op === SPMessage.OP_CONTRACT) {
       const value = rawSignedIncomingData(parsedValue)
       const authorizedKeys = Object.fromEntries(value.valueOf()?.keys.map(wk => {
-        const k = unwrapMaybeEncryptedData(wk)
+        const k = unwrapMaybeEncryptedDataOverride(wk)
```

Comment:
o4-mini (high) points out that that the returned instance will not use the `unwrapMaybeEncryptedDataOverride` function if/when `decryptedValue()` is called on it, but will use `unwrapMaybeEncryptedData`. Is that a problem? If so, can you update the code to save the function, and if not, could you add a comment explaining why it's OK?